AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -641,27 +654,51 @@ private void UnZipAndProcess(string filename, ArchiveFileType archiveFileType, b
 
             try
             {
-                IEnumerable<FileEntry> files = Extractor.ExtractFile(filename);
+                IEnumerable<FileEntry> files = Extractor.ExtractFile(filename).Where(x => x != null);
```

Comment:
What's the benefit on like 657?  I don't think you would get back a FileEntry object in the files that was null unless it throw an exception anyway.

Reply:
I'm not sure exactly how, but I was getting null FileEntries back in some cases. There shouldn't be any benefit except we can't 100% trust Extractor to not return nulls.